app/main/views/services.py

```diff
@@ -248,3 +248,38 @@ def get_archived_service(archived_service_id):
     ).first_or_404()
 
     return jsonify(services=service.serialize())
+
+
+@main.route(
+    '/services/<string:service_id>/status/<string:status>',
```

Comment:
Other than it means we have to wrap the line so it's not as easy to scan down the file for routes, no. It's purely a visual thing.